src/sentry/api/endpoints/project_group_index.py

```diff
@@ -324,7 +334,11 @@ def put(self, request, project):
         else:
             group_list = None
 
-        serializer = GroupSerializer(data=request.DATA, partial=True)
+        serializer = GroupValidator(
```

Comment:
probably should rename this everywhere, but it was especially confusing here because of StreamGroupSerializer